test/e2e/scaffold/apisix.go

```diff
@@ -139,6 +146,25 @@ func (s *Scaffold) apisixServiceURL() (string, error) {
 	return "", errors.New("no http port in apisix service")
 }
 
+func (s *Scaffold) apisixServiceHttpsURL() (string, error) {
+	if len(s.nodes) == 0 {
+		return "", errors.New("no available node")
+	}
+	var addr string
+	for _, node := range s.nodes {
+		if len(node.Status.Addresses) > 0 {
+			addr = node.Status.Addresses[0].Address
+			break
+		}
+	}
+	for _, port := range s.apisixService.Spec.Ports {
+		if port.Name == "https" {
+			return net.JoinHostPort(addr, strconv.Itoa(int(port.NodePort))), nil
+		}
+	}
+	return "", errors.New("no http port in apisix service")
```

Comment:
typo: should be "no https port defined in apisix service".

Reply:
Done